src/AdminService.Data.Provider.MsSql.Ef/AdminServiceDbContext.cs

```diff
@@ -11,6 +13,10 @@ public AdminServiceDbContext(DbContextOptions<AdminServiceDbContext> options)
     {
     }
 
+    public IQueryable<DbServiceConfiguration> ServicesConfigurations => _servicesConfigurations.AsQueryable();
+
+    private DbSet<DbServiceConfiguration> _servicesConfigurations { get; set; }

```

Comment:
move it to lines 11-13, private to the 11'th line

Reply:
fixed